submissions/ireneNakiyingi/database/transportationdb.sql

```diff
@@ -25,18 +23,17 @@ number_of_parents_children INT NOT NULL,
 number_of_siblings_spouses INT NOT NULL,
 point_of_embarkation VARCHAR(100) NOT NULL,
 PRIMARY KEY (id),
-FOREIGN KEY (id) REFERENCES registered_passengers(id)
+FOREIGN KEY (id) REFERENCES travel_id
```

Comment:
this is still wrong.
it should be FOREIGN KEY travel_id REFERENCES registered_passengers (id)

In english this means the column travel_id references the id column in the table registered_passengers.